simple_network_sim/network_of_populations.py

```diff
@@ -251,11 +393,21 @@ def getInternalInfection(dictOfStates, ageMixingInfectionMatrix, time, contactsM
 
 
 # CurrentlyInUse
-# internalStateDict should have keys like (age, compartment)
-#  The values are number of people in that node in that state
-#  diseaseProgressionProbs should have outward probabilities per timestep (rates)
-#  So we will need to do some accounting
 def internalStateDiseaseUpdate(currentInternalStateDict, diseaseProgressionProbs):
+    """Update the status of exposed individuals, moving them into the next disease state with a
+    probability defined in the given progression matrix.
```

Comment:
I think it's worth mentioning that this function returns a dictionary with new state, rather than updating things in-place.